Dockerfile

```diff
@@ -48,6 +49,8 @@ RUN pip install -U pip setuptools==58.3.0 && \
     pip install -q -r requirements.txt --compile --no-cache-dir
     # && apk del .build-deps
 
+RUN if [[ -z "$DEEPCHECKS_CI_TOKEN" ]] ; then true ; else pip install -q -r addon-requirements.txt --compile --no-cache-dir ; fi
```

Comment:
then true?

Reply:
basically skip if no ci token (for oss). I think i'll change it to use another env like (IS_DEEPCHEKS_OSS) so we won't have silent errors

Reply:
yea but it's weird, I'd try something like
```
if [[ -v DEEPCHECKS_CI_TOKEN ]] ; then
   pip install...
fi
```